houston/controller/hook/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * houston/controller/hook/index.js
+ * Handles all outside service inputs
+ *
+ * @exports {Object} - Koa router
+ */
+
+import Router from 'koa-router'
+
+import github from './github'
+import jenkins from './jenkins'
+
+const route = new Router({
+  prefix: '/hook'
+})
+
+route.use(github.routes(), github.allowedMethods())
+route.use(jenkins.routes(), jenkins.allowedMethods())
+
+// Use event hook listeners as well
+require('./atc')
```

Comment:
It loads all the websocket routes for atc. Technically it's not a koa route, but it still does the same type of logic a controller does so it makes sense being here. It also doesn't export anything which is why it's just a `require()` instead of `import`.